src/suite.py

```diff
@@ -40,662 +43,629 @@
-    features = ["perspective_score","stanford_polite"]
 
-    val = rescore(text,features,0)
-    predict = model.predict([val])[0]
-    logging.info("predicted "+str(predict)+" from "+str(val))
+counter = pickle.load(open("pickles/github_words.p", "rb"))
+our_words = dict([(i, word_frequency(i, "en") * 10**9) for i in counter])
+different_words = log_odds(
+    defaultdict(int, counter), defaultdict(int, our_words))
 
-    return [predict, val[0], val[1]]
+# logging.info("different_words"+str(different_words))
 
-# postprocessing (usually only done for toxic comments)
-# returns list of clean text variants
-def clean_text(text):
-    result = []
-    words = text.split(" ")
-    words = [a.strip(',.!?:; ') for a in words]
 
-    words = list(set(words))
-    words = [word for word in words if not word.isalpha() or word.lower() in different_words]
+# returns [isToxic, perspective_score, stanford_polite_score]
+def score_toxicity(text, model):
+  logging.info("get_prediction " + text)
+  features = ["perspective_score", "stanford_polite"]
 
-    for word in set(words):
-        # Maybe unkify?
-        result += [re.sub(r'[^a-zA-Z0-9]' + re.escape(word.lower()) + r'[^a-zA-Z0-9]', ' potato ', " "+text.lower()+" ").strip()]
+  val = rescore(text, features, 0)
+  predict = model.predict([val])[0]
+  logging.info("predicted " + str(predict) + " from " + str(val))
 
-    tokenizer = RegexpTokenizer(r'\w+')
-    all_words = tokenizer.tokenize(text)
-    # logging.info("all_words "+str(all_words))
-    # Try removing all unknown words
-    for word in set(all_words):
-        if word.lower() not in counter and word_frequency(word.lower(), "en") == 0 and len(word) > 2:
-            text = text.replace(word, '')
+  return [predict, val[0], val[1]]
 
-    result += [text]
-    return result
 
-def get_prediction(text,model):
-    logging.info("get_prediction "+text)
-    features = ["perspective_score","stanford_polite"]
+# postprocessing (usually only done for toxic comments)
+# returns list of clean text variants
+def clean_text(text):
+  result = []
+  words = text.split(" ")
+  words = [a.strip(",.!?:; ") for a in words]
+
+  words = list(set(words))
+  words = [
+      word for word in words
+      if not word.isalpha() or word.lower() in different_words
+  ]
+
+  for word in set(words):
+    # Maybe unkify?
+    result += [
+        re.sub(r"[^a-zA-Z0-9]" + re.escape(word.lower()) + r"[^a-zA-Z0-9]",
+               " potato ", " " + text.lower() + " ").strip()
+    ]
+
+  tokenizer = RegexpTokenizer(r"\w+")
+  all_words = tokenizer.tokenize(text)
+  # logging.info("all_words "+str(all_words))
+  # Try removing all unknown words
+  for word in set(all_words):
+    if word.lower() not in counter and word_frequency(
+        word.lower(), "en") == 0 and len(word) > 2:
+      text = text.replace(word, "")
+
+  result += [text]
+  return result
+
+
+def get_prediction(text, model):
+  logging.info("get_prediction " + text)

```

Comment:
hmm. logs shouldn't store user data. if you want to store comment identifiers, than that's fine